### DISCUSSION TITLE
more verbose PR

### DISCUSSION THREAD

**Discussion by dozer111** [rectorphp/rector#8619] [General]
Hi all. I love rector. This is cool lib, and as for me your release notes are pretty verbose

But

lets take https://github.com/rectorphp/rector/releases/tag/0.19.3 as example

---

What do i see?

- no new rules
- new command
- new ConfigBuilder
- ...

but, what does mean "New Config builder"?

We are going to link(cool that you are leave it) https://github.com/rectorphp/rector-src/pull/5503 and what do we see - nothing

 For me, as for user who does not know the internals well, its a big question

Does this feature is something that i need? Or maybe it is something about cool internal changes? IDK

Those "features" which you mark as something cool are clear only for you as contributors, but not clear for me as user

---

Can u do you PR much verbose to much more easy understand why some code was added and what does it mean to person who use it ?

**Comment by TomasVotruba** [comment]
Hi,

thanks for feedback. We don't use PRs for docummentation, as we have other sources that we maintain. We also use PRs to test features in the dev-wild, before we're sure to put them into final release :)

* https://github.com/rectorphp/rector/releases - best way is to subsribe to releases
* https://twitter.com/rectorphp
* https://getrector.com/documentation

I recommend following those to get up to date into of what's new in Rector :+1:
